index.js

```diff
@@ -94,7 +92,13 @@ module.exports = function (logger) {
         }
       })
     }
-    logger.error()
+
+    logger.error('\nUnknown handles:\n')
```

Comment:
I guess this should be done outside function passed to `known.forEach`. Currently this code repeats all unknown handles after each known handle.